### PR TITLE
Full content re-write for legacy become-editor page

### PR DESCRIPTION
**Pull request by gtdata**
https://trello.com/c/PBEqgI5y/232-review-content-for-become-an-editor-legacy

### CODE REVIEW

File: becoming_an_editor_or_admin.md [hunk 1]
```diff
@@ -1,75 +1,66 @@
...
-* creating and editing data records on data.gov.uk
+## Permissions explained
 
-If you’re an editor, you’ll be responsible for:
+Admins can also:
 
-* creating and editing data records on data.gov.uk
+* approve editor requests
+* respond to FOI requests (unless otherwise specified by the publisher)
+* create new editors or admins
```
Comment (andrewgarner) [inline becoming_an_editor_or_admin.md]: This is duplicating the "adding editor and admin accounts" item in the previous list. This list is introduced as "can also" so it doesn't quiet make sense.

Maybe we could combine both lists?

File: becoming_an_editor_or_admin.md [hunk 2]
```diff
@@ -1,75 +1,66 @@
...
-You need to ask for an administrator or editor account. There’s no limit on how many administrators or editors your organisation can have.
+Editors can also:
 
+* manage and configure the harvesting for their organisation
 
-### Administrator
+## How to get an account
 
-The administrator (admin) account serves to manage all editors creating/editing metadata under a given parent department. Administrators can create editors within their publisher hierarchy or across the family of publishers.
+1. Go to <http://data.gov.uk/user/register>. Choose a username and password to complete the registration. Click 'Create new account'.
```
Comment (andrewgarner) [inline becoming_an_editor_or_admin.md]: We should link to the secure version of this page "https://data.gov.uk/user/register"

**Review by andrewgarner — CHANGES_REQUESTED** [review]

File: becoming_an_editor_or_admin.md (hunk 2 above)
Comment (gtdata) [reply becoming_an_editor_or_admin.md]: good point, will update

File: becoming_an_editor_or_admin.md (hunk 1 above)
Comment (gtdata) [reply becoming_an_editor_or_admin.md]: makes sense, will update, thx

**Review by andrewgarner — APPROVED** [review]

File: becoming_an_editor_or_admin.md [hunk 3]
```diff
@@ -1,75 +1,63 @@
...
-* appointing other users as editors and other administrators
-* updating details about your organisation on data.gov.uk
-* creating and editing data records on data.gov.uk
+Admins can also:
 
-If you’re an editor, you’ll be responsible for:
+* create and add new editors or admins
+* approve editor requests
+* manage the activity of editors
+* respond to FOI requests (unless otherwise specified by the publisher)
```
Comment (laurentqro) [inline becoming_an_editor_or_admin.md:17]: should we specify the meaning of the FOI acronym the first time it's used on the page?

Comment (gtdata) [reply becoming_an_editor_or_admin.md:17]: All the guidance will be reviewed & rewritten after launch, so this version is only meant to be a quick re-write. But, it's such a small edit, I think it's worth doing. Will update.

Comment (andrewgarner) [reply becoming_an_editor_or_admin.md:17]: @gtdata https://github.com/datagovuk/guidance/edit/gh-pages/becoming_an_editor_or_admin.md

**Pull request merged**